### PR TITLE
Fix external URL params

### PR DESCRIPTION
**Pull request by richaagarwal**
details [here](https://github.com/cfpb/cfgov-refresh/pull/1780) bc github doesn't know how to properly reopen pull requests 👿

### CODE REVIEW

File: cfgov/jinja2/v1/external-site/index.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 {% set request_host = 'http://' + request.META.HTTP_HOST %}
 {% set request_referrer = request.META.HTTP_REFERER or request_host %}
 
-{% set request_url_param = request.GET.get('ext_url', '') %}
-{% set external_url = request_url_param or request_host %}
+{% set request_url = request.get_full_path()[24:] %}
```
Comment (kurtw) [inline cfgov/jinja2/v1/external-site/index.html]: What's with the [24:]? How can we be sure that's the correct length?

Comment (richaagarwal) [inline cfgov/jinja2/v1/external-site/index.html]: yeah sorry should've been more clear. i'll add in a comment, one sec

**Comment by richaagarwal** [comment]
added a comment. i looked around and there were no other jinja filters that would've done this in a cleaner way, and there were no other methods on request that get us the data we want. selecting request.GET.get('ext_url') only gives us back part of the params. 

i COULD write a filter to make this cleaner, but it's sort of the same thing either way.

File: cfgov/jinja2/v1/external-site/index.html [hunk 2]
```diff
@@ -2,8 +2,14 @@
 {% set request_host = 'http://' + request.META.HTTP_HOST %}
 {% set request_referrer = request.META.HTTP_REFERER or request_host %}
 
-{% set request_url_param = request.GET.get('ext_url', '') %}
-{% set external_url = request_url_param or request_host %}
+
+{% set request_url = request.get_full_path()[24:] %}
+{# request.get_full_path always begins with "/external-site/ext_url=",
```
Comment (kurtw) [inline cfgov/jinja2/v1/external-site/index.html]: Isn't it "/external-site/?ext_url="?

**Comment by kurtw** [comment]
Whoops commented on the wrong PR. LGTM :+1:

**Comment by sebworks** [comment]
:+1: , should we be using url encoding in the processors when building the links?

**Pull request merged**